Fixed a few typos. (#25395)
mdn/content: files/en-us/web/css/transition-duration/index.md

{{EmbedInteractiveExample("pages/css/transition-duration.html")}}

> You may specify multiple durations; each duration will be applied to the corresponding property as specified by the {{ cssxref("transition-property") }} property, which acts as a master list. If there are fewer durations specified than in the master list, the user agent repeats the list of durations. If there are more durations, the list is truncated to the right size. In both cases, the CSS declaration stays valid.

Reviewer: ```suggestion
You may specify multiple durations; each duration will be applied to the corresponding property as specified by the {{ cssxref("transition-property") }} property, which acts as a master list. If the number of specified durations is less than in the master list, the user agent repeats the list of durations. If the number of specified durations is more than in the master list, the list is truncated to the right size. In both the cases, the CSS declaration stays valid.
```

Author: These edits are great: they make the text easier to read and understand. I would only say that "In both the cases" sounds off to me. I'd leave it as "In both cases".

Reviewer: Great, you can then push a new commit with these changes (minus the last "the").
